pkg/engine/executor/dataobjscan_test.go

```diff
@@ -20,7 +21,6 @@ import (
 var (
 	labelMD    = buildMetadata(types.ColumnTypeLabel)
 	metadataMD = buildMetadata(types.ColumnTypeMetadata)
-	builtinMD  = buildMetadata(types.ColumnTypeBuiltin)
```

Comment:
Should we add `timestampMD` and `messageMD` here? It's a bit inconsistent now

Reply:
After talking online, being consistent in either direction is going to be a bit awkward for now. I'm ok leaving this as-is